app/Models/Forum/Forum.php

```diff
@@ -216,4 +216,16 @@ public function isOpen()
     {
         return $this->forum_type === 1;
     }
+
+    public function toMetaDescription()
+    {
+        $stack = [__('forum.title')];
```

Comment:
I don't think we use `__` anywhere else in the codebase, probably better to remain consistent and use `trans`.